kvbc/include/merkle_processor/IMerkleProcessor.h

```diff
@@ -0,0 +1,52 @@
+// Concord
+//
+// Copyright (c) 2023 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the sub-component's license, as noted in the
+// LICENSE file.
+
+#pragma once
+
+//#include <stdint.h>
+#include <string>
+#include <vector>
+#include "categorization/updates.h"
+#include "proof_processor/IProofProcessor.h"
+#include "util/sliver.hpp"
+
+namespace concord {
+namespace kvbc {
+namespace sparse_merkle {
+
+// This is an interface used for managing a set of Merle trees' storage.
```

Comment:
So shouldn't its name reflect that?

Reply:
It is named IMerkleProcessor because the proofs it will maintain and provide are based on Merkle paths.